services/app-web/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -58,37 +82,43 @@ export const Dashboard = (): React.ReactElement => {
     }
 
     if (loggedInUser.__typename !== 'StateUser') {
-        return <div>CMS Users not supported yet.</div>
+        return <div id="dashboard-page" className={styles.wrapper}><div>CMS Users not supported yet.</div> </div>
     }
 
+
     const programs = loggedInUser.state.programs
+    const submissionRows: SubmissionInDashboard[] = []
 
-    const submissionRows = data.indexSubmissions.edges
+    data?.indexSubmissions2.edges
         .map((edge) => edge.node)
-        .map((row) => {
-            if (row.__typename === 'DraftSubmission') {
-                return {
-                    __typename: row.__typename,
-                    id: row.id,
-                    name: row.name,
-                    programIDs: row.programIDs,
-                    updatedAt: row.updatedAt,
-                    submissionType: row.submissionType,
-                }
-            } else {
-                return {
-                    __typename: row.__typename,
-                    id: row.id,
-                    name: row.name,
-                    programIDs: row.programIDs,
-                    submittedAt: row.submittedAt,
-                    updatedAt: row.updatedAt,
-                    submissionType: row.submissionType,
-                }
+        .forEach((sub) => {
+            const currentRevision = sub.revisions[0]
+            const currentSubmissionData = base64ToDomain(
+                currentRevision.revision.submissionData
+            )
+            console.log('IN HERE')
+            if (currentSubmissionData instanceof Error) {
```

Comment:
This is a good move, we don’t want to break the dashboard just b/c a row has old submissionData in it. We’ll solve this better one day but for now just hiding those rows seems good to me.